github/provider.go

```diff
@@ -26,6 +26,26 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("GITHUB_OWNER", nil),
 				Description: descriptions["owner"],
 			},
+			"retryable_errors": {
+				Type:     schema.TypeList,
+				Elem:     &schema.Schema{Type: schema.TypeInt},
+				Optional: true,
+				DefaultFunc: func() (interface{}, error) {
+					defaultErrors := []int{500, 502, 503, 504}
+					errorInterfaces := make([]interface{}, len(defaultErrors))
+					for i, v := range defaultErrors {
+						errorInterfaces[i] = v
+					}
+					return errorInterfaces, nil
+				},
+				Description: descriptions["retryable_errors"],
+			},
+			"max_retries": {
```

Comment:
Does it make sense to set this one to 0 by default? 
The need for this functionality is to gracefully handle transient failures that happens quite frequently but I feel like setting this to 0, despite retaining the current behaviour, is probably not what we want.
Just my 2 cent

Reply:
Should I set it to 3?

Reply:
@ilmax I have changed it to 3, let me know what you think

Reply:
3 sounds reasonable to me, thank you.